### ISSUE TITLE
[BUG_SP]JD 7R with Laforge EZ 1700 not performing turns properly

### ISSUE THREAD

**Issue by SavageMkII** [Courseplay/Courseplay_FS22#828]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Generate course on field 70 of Elmcreek(slightly customized field shape) with 2 headlands, corners set to sharp. Run tractor with spreader on course to spread lime on field. Observe 

**Actual behavior**
Upon reaching the sharp corners of the course, the vehicle begins by reversing, but not far enough, then moves forward and around the turn, but also not far enough, then proceeds to reverse in circles until manually stopped. As for the row end turns, the generated turn maneuver does not line up with the next row properly and leaves a small patch of field uncovered. 

**Expected behavior**
The vehicle should perform the turn properly and continue on the course, without missing any of the field.

**Screenshots**
![fsScreen_2022_02_01_15_59_11](https://user-images.githubusercontent.com/25916123/152067458-7ddfa2e1-8d18-46cc-89e8-1ba4483f3abf.png)
![fsScreen_2022_02_01_15_56_15](https://user-images.githubusercontent.com/25916123/152067510-0d304aaf-236b-4569-a1db-927720313c54.png)
![fsScreen_2022_02_01_16_04_57](https://user-images.githubusercontent.com/25916123/152067517-239fb97a-91c7-4c93-aef9-b5030fa88e0a.png)
![fsScreen_2022_02_01_16_05_00](https://user-images.githubusercontent.com/25916123/152067522-efb4c919-b4f6-43b7-8d0e-6e24f9191b25.png)

**Courseplay version**
Version used: 7.0.0.25 1c8a5d0

**Log**
(https://gist.github.com/SavageMkII/9b373e33daeae959a634fed18de98629).

**Map**
Vanilla Elmcreek

**Vehicles/ implements involved**
Which vehicle or implement? Vanilla John Deere 7R with Bredal K165(6m spread unit)
Mod yes/no ? No

**Relevant script mods**
AD, EV, Lock Steering Axles

**Comment by SavageMkII** [comment]
To be more clear, I have the lock steering axles mod(by Ifko[nator], and tested with the steering axle on the spreader locked and unlocked, and it didn't make an appreciable difference. I also tried adjusting the turn radius setting and the vertical offset of the tool, also with little to no effect.

**Comment by pvaiko** [comment]
I can't reproduce this and you have most of the debug channels turned off. Make sure to have the debug enabled on the vehicle, and channels fieldwork, pathfinder, reverse, ppc, turn, courses are all enabled.

**Comment by SavageMkII** [comment]
Hi Peter, thanks for looking into this. I should have done some more testing before opening this issue. I've found the culprit:
![fsScreen_2022_02_02_15_55_23](https://user-images.githubusercontent.com/25916123/152251873-c0e3edf2-db83-4157-9af9-d53c1eab85e6.png)
If I remove this weight from the tractor, everything works perfectly. I did some more testing with the debug channels you wanted, but I think this is going to be a low priority issue, so I will close for now.
